_opensearch/snapshots/snapshot-management.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: default
+title: Snapshot management
+parent: Snapshots
+nav_order: 20
+has_children: false
+---
+
+# Snapshot management
```

Comment:
Please call out the defaults in Snapshot management, example - default snapshot name, snapshot creation/deletion schedule, etc. @bowenlan-amzn can you pls help provide those?